.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 # cancel already running builds of the same branch or pull request
 concurrency:
-  group: ci-${{ github.head_ref }} || concat(${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

Comment:
I've changed this because the codeql job cancelled itself, considering the build job a higher priority